index.html

```diff
@@ -74,11 +78,30 @@ <h1 id="ctl-messages">Messages Exchanged</h1>
       <h4>messages exchanged since August 1, 2013<h4>
     </div>
   </div>
+  <div class="section slide-tmi">
+    <div class="wrapper">
+      <img src="/img/tmi-logo.png" width="256px" height="256px" />
+      <h1>$850,000</h1>
+      <h3>Donated to DoSomething</h3>
```

Comment:
Since these are both part of the same "sentence", I think i'd recommend putting them both in one heading tag, and using something like `span` or `strong` to emphasize & style the dollar amount.